File: Seed/Java/src/main/java/Hayaa/ConfigSeed/AppConfig.java
package Hayaa.ConfigSeed;

import hayaa.basemodel.model.BaseData;

import java.io.Serializable;
import java.util.List;
import java.util.UUID;

class AppConfig extends BaseData implements Serializable {

    private Integer appId;
    private Integer appConfigId;
    private String solutionID;
    private String solutionName;
    private String configContent;
    private Integer version;
    private List<ComponentConfig> components;

    public Integer getAppId() {
        return appId;
    }

    public void setAppId(Integer appId) {
        this.appId = appId;
    }

    public Integer getAppConfigId() {
        return appConfigId;
    }

    public void setAppConfigId(Integer appConfigId) {
        this.appConfigId = appConfigId;
    }

    public String getSolutionID() {
        return solutionID;
    }

    public void setSolutionID(String solutionID) {
        this.solutionID = solutionID;
    }

    public String getSolutionName() {
        return solutionName;
    }

    public void setSolutionName(String solutionName) {
        this.solutionName = solutionName;
    }

    public String getConfigContent() {
        return configContent;
    }

    public void setConfigContent(String configContent) {
        this.configContent = configContent;
    }

    public Integer getVersion() {
        return version;
    }

    public void setVersion(Integer version) {
        this.version = version;
    }

    public List<ComponentConfig> getComponents() {
        return components;
    }

    public void setComponents(List<ComponentConfig> components) {
        this.components = components;
    }





}

File: Service/Hayaa.RemoteConfig.ServiceV2/Model/AppConfig.cs
﻿using Hayaa.BaseModel;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.RemoteConfig.Service
{
    /// <summary>
    /// 宿主程序配置方案
    /// </summary>
    [Serializable]
    public class AppConfig : BaseData
    {
        /// <summary>
        /// 配置数据ID
        /// </summary>	
        public int AppConfigId
        {
            get;
            set;
        }
        /// <summary>
        /// AppId
        /// </summary>
        public int AppId { set; get; }
        /// <summary>
        /// 解决方案ID
        /// </summary>	
        public String SolutionID
        {
            get;
            set;
        }
        /// <summary>
        /// 解决方案名称
        /// </summary>	
        public string SolutionName
        {
            get;
            set;
        }
        /// <summary>
        /// App级别配置内容
        /// </summary>	
        public string ConfigContent
        {
            get;
            set;
        }

        /// <summary>
        /// 程序配置方案版本 
        /// </summary>
        public int Version
        {
            get;
            set;
        }
        /// <summary>
        /// 程序组件配置内容
        /// </summary>
        public List<ComponentConfig> Components { get; set; }


    }
}
